mkosi/resources/man/mkosi.1.md

```diff
@@ -1154,8 +1154,14 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     used as an initramfs.
 
 `Ssh=`, `--ssh=`
-:   If specified, an **sshd** socket unit and matching service are installed
-    in the final image that expose SSH over VSock. When building with this
+:   Specifies whether to install an **sshd** socket unit and matching service
+    in the final image. Takes one of `always`, `never`, `auto` or `runtime`.
+    Defaults to `auto`. If set to `auto` and `systemd-ssh-generator`
```

Comment:
That's a bit dense, let's split this up slightly.
```suggestion
    in the final image. Takes one of `always`, `never`, `auto` or `runtime`.
    Defaults to `auto`. 
    
    If set to `auto` and `systemd-ssh-generator`
```

Also, to maybe make this clearer to users who might wondern about different packaging among distros
```suggestion
    in the final image. Takes one of `always`, `never`, `auto` or `runtime`.
    Defaults to `auto`.
    
    If set to `auto` and the generator binary `systemd-ssh-generator`
```